app/src/test/java/org/wikipedia/imagesearch/EncoderTest.java

```diff
@@ -0,0 +1,55 @@
+package org.wikipedia.imagesearch;
+
+import android.content.ContentResolver;
+import android.content.Context;
+import android.net.Uri;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+/**
+ * Created by amawai on 2018-04-13.
+ */
+
+public class EncoderTest {
```

Comment:
Is it possible to extend the EncoderTest to test edge cases?

Reply:
Addressed in commit f196158.